pkg/registry/core/pod/storage/eviction.go

```diff
@@ -149,16 +149,29 @@ func (r *EvictionREST) Create(ctx context.Context, obj runtime.Object, createVal
 		} else if len(pdbs) == 1 {
 			pdb := pdbs[0]
 			pdbName = pdb.Name
-			// Try to verify-and-decrement
-
-			// If it was false already, or if it becomes false during the course of our retries,
-			// raise an error marked as a 429.
-			if err := r.checkAndDecrement(pod.Namespace, pod.Name, pdb, dryrun.IsDryRun(deletionOptions.DryRun)); err != nil {
-				return err
-			}
+			refresh := false
+			refreshPDB := &pdb
```

Comment:
Why do you need this?
Instead two lines above you can simply do:

pdb := &pdbs[0]

Reply:
refresh can avoid unnecessary get request in case of not conflict